### ISSUE TITLE
Caching to improve performance

### ISSUE THREAD

**Issue by anojperera** [GSS-Cogs/dd-cms#687]
Investigate adding caching to either using NGINX proxy or varnish as a caching engine.

**Comment by JasonHowell** [comment]
Improve load times.
Baseline light house scores 
Enable Plone caching and re-baseline
Enable NGINX cachning and re-baseline

**Comment by anojperera** [comment]
Managed to get a good baseline performance. Noted a we have 400ms on JS loading this will be forced to cache.

![Screenshot 2022-12-06 at 14.12.13.png](https://images.zenhubusercontent.com/619f67e36a20872245102c1c/72366a53-3e33-44a6-b303-b5dd0cc9413c)

**Comment by anojperera** [comment]
Enable caching on the plone server.

**Comment by anojperera** [comment]
Incorporating some of the enhancements Rhys has made with caching on the front page we are able to reach...

For this test we have eliminated network latency all together by running everything locally.
![Screenshot 2022-12-21 at 11.28.17.png](https://images.zenhubusercontent.com/619f67e36a20872245102c1c/45c67936-8be7-4413-b552-2ff4cd4e56f9)

**Comment by eldeal** [comment]
This is blocked by #755 as we're looking to make this change once across all sites once we can move to a single scaled volto deployment. The work for this will actually be completed on the platform board as #83 so potentially we can just close this (or happy to keep it for recording when we can expect to see these improvements in action)

**Comment by JasonHowell** [comment]
On hold at the moment for a more sensible time to integrate.
